src/JavascriptVM/JavascriptVM.tsx

```diff
@@ -170,7 +170,59 @@ export const VMProvider: FunctionComponent = ({ children }) => {
     return () => {
       console.log("Destroying VMContext.Provider");
     };
-  });
+  }, [canvasRef]);
+
+  function setSimulatorCameraMode(
+    simulator: Sim3D,
+    robotRef: Handles.RobotHandle,
+    cameraMode: CameraView
```

Comment:
There is a mismatch here with cameraMode and CameraView.
I think we should rename CameraView into CameraMode.

Reply:
The parameters here are slightly confusing! but i wont try to fix that in this PR